python/ray/__init__.py

```diff
@@ -33,6 +33,20 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if "psutil" in sys.modules:
+    print("Not using bundled version of psutil")
+else:
+    psutil_path = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                        "psutil_files")
+    sys.path.insert(0, psutil_path)
+
+if "setproctitle" in sys.modules:
+    print("Not using bundled version of setproctitle")
+else:
+    setproc_path = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                        "setproctitle_files")
+    sys.path.insert(0, setproc_path)
+
```

Comment:
We don't need mess with the import path if we are directly import from `ray.thirdparty_files.psutil`